float32 linear_speed
float32 step
float32 angular_speed
int32 color
---
bool success